packages/firebase_remote_config/example/test_driver/firebase_remote_config_e2e.dart

```diff
@@ -25,14 +26,18 @@ void main() {
       expect(lastFetchTime.isBefore(DateTime.now()), true);
       await remoteConfig.fetch(expiration: const Duration(seconds: 0));
       expect(remoteConfig.lastFetchStatus, LastFetchStatus.success);
-      await remoteConfig.activateFetched();
+      final activated = await remoteConfig.activateFetched();
+      expect(activated, true);
 
+      // TODO should verify that our config settings actually took
```

Comment:
Just a note, there were problems here when I was testing and I could not figure them out. Apparently there was a remote-config bug in 7.1.0 (which I was testing) where remoteconfig was completely broken https://github.com/firebase/firebase-android-sdk/issues/2186 and remote settings did not come in. Bears a retest

Reply:
I got a successful run locally 👍